app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -3212,12 +3213,14 @@ class ChatActivity :
 
         try {
             require(fileUri.isNotEmpty())
-            UploadAndShareFilesWorker.upload(
+            upload(
                 fileUri,
                 room,
                 currentConversation?.displayName!!,
                 metaData
-            )
+            ) { showProgressBar: Boolean ->
+                binding.idprogressbar.visibility = if (showProgressBar) View.VISIBLE else View.GONE
```

Comment:
```suggestion
                binding.idprogressbar.visibility = if (showProgressBar) View.VISIBLE else {
                    chatViewModel.refreshChatParams(
                        setupFieldsForPullChatMessages(
                            true,
                            globalLastKnownFutureMessageId,
                            true
                        )
                    )
                    View.GONE
                } 
```

Breaks with the recent refactoring of `ChatActivity`. Insert this line to fix it, so that uploaded files show up after sending.